src/main/java/RacingGame/Domain/CarManager.java

```diff
@@ -1,35 +1,64 @@
 package RacingGame.Domain;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
 
 public class CarManager {
     List<Car> carList;
     int[][] carHistory;
 
-    public CarManager(int carNum, int trialNum) {
+    public CarManager(int trialNum, String[] carNames) {
         carList = new ArrayList<>();
-        carHistory = new int[carNum][trialNum];
-        for (int i = 0; i < carNum; i++) {
-            Car car = new Car();
+        carHistory = new int[carNames.length][trialNum];
+        for (int i = 0; i < carNames.length; i++) {
+            Car car = new Car(carNames[i]);
             carList.add(car);
         }
     }
```

Comment:
fori 말고 이런 방식을 사용해도 좋을것 같아요

```suggestion
        for (String carName : carNames) {
            Car car = new Car(carName);
            carList.add(car);
        }
```